pb/gen.sh

```diff
@@ -5,4 +5,4 @@
 
 # You might need to go get -v github.com/gogo/protobuf/...
 
-protoc --gofast_out=plugins=grpc:. --gofast_opt=paths=source_relative -I=. badgerpb3.proto
+protoc --gogofaster_out=plugins=grpc:. --gogofaster_opt=paths=source_relative -I=. badgerpb3.proto
```

Comment:
Do we need this grpc plugin? Can we remove it?

Reply:
No, we don't need. Yes, we can remove it. Removed.